cmd/migrate/migrate.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/ledgerwatch/bolt"
+	"github.com/ledgerwatch/turbo-geth/common/dbutils"
+)
+
+func encodeToCBOR(buf []byte) ([]byte, error) {
+	acc, err := decodeRLP(buf)
+
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]byte, acc.EncodingLengthForStorage())
+	acc.EncodeForStorage(res)
+	return res, nil
+}
+
+// This method does not Commit
+func convertDatabaseToCBOR(db *bolt.DB, maxTxOperations uint) error {
+	var lastKey []byte = nil
+	var done bool = false
+	for !done {
+		var i uint
+		err := db.Update(func(readTx *bolt.Tx) error {
+			var accountBucket *bolt.Bucket = readTx.Bucket(dbutils.AccountsBucket)
+			var k, v []byte
+			c := accountBucket.Cursor()
+
+			for k, v = c.Seek(lastKey); k != nil; k, v = c.Next() {
+
+				enc, err := encodeToCBOR(v)
+				if err != nil {
+					return err
+				}
+				i++
+				lastKey = k
+				err = accountBucket.Put(k, enc)
+				if err != nil {
+					return err
+				}
+				if i+1 == maxTxOperations {
+					lastKey = k
+					break
+				}
+			}
+
+			if k == nil {
+				done = true
+			}
+			return readTx.Commit()
```

Comment:
Oh, and you don't need this `Commit` call here, `Update` function commits automatically if there are not errors. And, given that, name `readTx` becomes a bit misleading. Better to just use `tx`